SDK Showcase/SDK Showcase/Model/SDKConfigModel.swift

```diff
@@ -13,10 +13,11 @@ struct SDKConfigModel {
     let appScheme: String? = nil
     
     /// Default config model
-    static let `default` = SDKConfigModel(dictionary: ["ONGAppIdentifier": "ExampleApp",
-                                                       "ONGAppPlatform": "ios",
-                                                       "ONGAppVersion": "6.2.4",
-                                                       "ONGAppBaseURL": "https://mobile-security-proxy.onegini.com",
-                                                       "ONGResourceBaseURL": "https://mobile-security-proxy.onegini.com/resources/",
-                                                       "ONGRedirectURL": "oneginiexample://loginsuccess"])
+    static let defaultDictionary = ["ONGAppIdentifier": "SDKShowCase",
+                                                "ONGAppPlatform": "ios",
+                                                "ONGAppVersion": "1.0.0",
```

Comment:
`1.0.0` before any release? How about `0.x.y`?

Reply:
yes, 1.0.0 is very fine here